src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/FullscreenTextResource.svelte

```diff
@@ -92,12 +95,24 @@
                     </button>
                 </div>
             </div>
-            <div class="prose mx-auto overflow-y-scroll px-4 pb-4 md:px-0 {!!resource?.audioUrl && 'mb-14'}">
-                {#if resource?.html}
-                    {@html resource.html}
-                {:else}
-                    {$translate('page.passage.resourcePane.error.value')}
+            <div class="self-center overflow-y-scroll">
+                {#if resource?.communityEdition}
+                    <div class="float-right p-4 pe-5 text-warning">
```

Comment:
Does `float-right` behave as expected for RTL?

Reply:
Great question, I will test that

Reply:
It needs to be `float-end`, thanks for catching!